frog/imports/api/activities.js

```diff
@@ -20,7 +20,7 @@ export const addActivity = (
   groupingKey: ?string
 ) => {
   if (id) {
-    const toSet = omitBy({ data, groupingKey }, isNil);
+    const toSet = omitBy({ activityType, data, groupingKey }, isNil);
```

Comment:
Why this?

Reply:
It makes thing a bit easier, in the chooseActivity select function, the DB is directly modified through `Activities.update(this.props.activity._id, $set: { activityType: activityType.id }});` since the add activity previously could not change the type of an activity. But I think it makes more sense to use the `addActivity` function with an existing id to modify it, no ?